hs_core/tests/api/native/test_folder_download_zip.py

```diff
@@ -24,39 +24,57 @@ def setUp(self):
             groups=[]
         )
 
+        self.res = create_resource(resource_type='CompositeResource',
+                                   owner=self.user,
+                                   title='Test Resource',
+                                   metadata=[])
+
+        ResourceFile.create_folder(self.res, 'foo')
+
         # create files
         self.n1 = "test1.txt"
-
         test_file = open(self.n1, 'w')
         test_file.write("Test text file in test1.txt")
         test_file.close()
 
-        test_file = open(self.n1, "r")
+        self.test_file = open(self.n1, "r")
+        add_resource_files(self.res.short_id, self.test_file, folder='foo')
 
-        self.res = create_resource(resource_type='GenericResource',
-                                   owner=self.user,
-                                   title='Test Resource',
-                                   metadata=[], )
-
-        ResourceFile.create_folder(self.res, 'foo')
+        # copy refts file into new file to be added to the resource as an aggregation
+        reft_data_file = open('hs_core/tests/data/multi_sites_formatted_version1.0.refts.json', 'r')
+        refts_file = open('multi_sites_formatted_version1.0.refts.json', 'w')
+        refts_file.writelines(reft_data_file.readlines())
+        refts_file.close()
+        self.refts_file = open('multi_sites_formatted_version1.0.refts.json', 'r')
 
-        # add one file to the resource
-        add_resource_files(self.res.short_id, test_file, folder='foo')
+        add_resource_files(self.res.short_id, self.refts_file)
+        self.res.create_aggregation_xml_documents()
 
     def tearDown(self):
         super(TestFolderDownloadZip, self).tearDown()
         if self.res:
             self.res.delete()
+        if self.test_file:
+            os.remove(self.test_file.name)
+        if self.refts_file:
+            os.remove(self.refts_file.name)
         GenericResource.objects.all().delete()
         istorage = IrodsStorage()
-        if istorage.exists(self.output_path):
```

Comment:
@sblack-usu It'd be better to make ```istorage``` a class level variable in setup() so that it can be referenced here in tearDown() rather than creating a new IrodsStorage() object in tearDown()